mathics_scanner/data/operators.yml

```diff
@@ -5284,7 +5311,7 @@ PlusMinus:
 
 Postfix:
   actual-precedence: 640
-  precedence: 640
+  precedence: 70  # Seems weird but this is what old Mathics code has
```

Comment:
In WL it is 70, so I think it is correct

Reply:
I will remove the comment then. Thanks!